src/index.html

```diff
@@ -7,6 +7,302 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#">
+        <img
+        src="./images/logo.png"
+        alt="MOYO logotype"
+        class="header__img">
```

Comment:
```suggestion
        <img
          src="./images/logo.png"
          alt="MOYO logotype"
          class="header__img"
        >
```
Pay attention to formatting